ledger/participant-integration-api/src/main/scala/platform/store/dao/events/EventsTableOracle.scala

```diff
@@ -157,17 +160,19 @@ object EventsTableOracle extends EventsTable {
       "exercise_argument_compression" -> argumentCompression,
       "exercise_result" -> result,
       "exercise_result_compression" -> resultCompression,
-      "exercise_actors" -> event.actingParties.toArray[String],
+      "exercise_actors" -> event.actingParties.toJson.compactPrint,
       "exercise_child_event_ids" -> event.children
         .map(EventId(transactionId, _).toLedgerString)
-        .toArray[String],
+        .toList
+        .toJson
+        .compactPrint,
     ) ++ emptyCreateFields
 
   private val emptyCreateFields = Vector[NamedParameter](
     "create_argument" -> Option.empty[Array[Byte]],
     "create_argument_compression" -> Option.empty[Short],
-    "create_signatories" -> Option.empty[Array[String]],
-    "create_observers" -> Option.empty[Array[String]],
+    "create_signatories" -> "[]",
```

Comment:
Why empty array string over Option.empty?

Reply:
Because we're treating the CLOB fields as a JSON field. On line 85 of this file, we're rendering submitters as 
`"submitters" -> submitterInfo.map(_.actAs).getOrElse(List.empty).toJson.compactPrint,`. Later when we access this field, we're expecting it to be a JSON array. I'm not sure this changes anything - i.e. I'm not sure that it would fail if there were simply an empty string, but I'm biased towards consistency.